incubator/binding-risingwave/src/test/java/io/aklivity/zilla/runtime/binding/risingwave/internal/config/RisingwaveOptionsConfigAdapterTest.java

```diff
@@ -82,7 +82,17 @@ public void shouldReadOptions()
                         ]
                       }
                     }
-                  }
+                  },
+                  "udf": [
+                    {
+                      "server": "http://udf.zillabase.dev:8815",
+                      "language": "java"
+                    },
+                    {
+                      "server": "http://udf-python.zillabase.dev:8815",
+                      "language": "python"
+                    }
+                    ]
```

Comment:
Please include `udf` in `shouldWriteOptions` test method implementation as well.